components/application/index.jsx

```diff
@@ -33,15 +32,17 @@ const Application = ({
         <h2 className="sr-only">Navigate your data</h2>
         {dataProvider && (
           <ActivitySelect>
-            {activities.map(({ id, path }) => (
-              <ActivitySelect.Option
-                dataProvider={dataProvider.id}
-                key={id}
-                value={id}
-                selected={id === activity.split('/')[0]}
-                path={path}
-              />
-            ))}
+            {activities.routes
+              .filter((activity) => activity.parent == null)
+              .map(({ path, test }) => (
```

Comment:
This exposed activities configuration internals. I do not know how to do this better at the moment.

Reply:
I don't see a problem with it.